libraries/AP_Scripting/examples/RM3100_self_test.lua

```diff
@@ -0,0 +1,107 @@
+-- Runs the Built-In Self Test on the RM3100 LR circuits
+-- Note COMPASS_DISBLMSK should have the 16th bit set to 1 (RM3100)
+
+-- Init RM3100 on bus 0
+local rm3100 = i2c:get_device(0, 0x20)
+assert(rm3100 ~= nil, "i2c get_device error, cannot run RM3100 self test")
+
+-- Queues a Built-In Self Test
+function queue_test()
+    gcs:send_text('1', "Running RM3100 self test")
```

Comment:
```suggestion
    gcs:send_text(1, "Running RM3100 self test")
```

Same in other places.